lib/rets/client.rb

```diff
@@ -9,11 +9,13 @@ class Client
 
     def initialize(options)
       @options = options
-      @capabilities = nil
       clean_setup
     end
 
     def clean_setup
+      if options.fetch(:login_after_error, false)
```

Comment:
Ah I should have explained myself. My thinking here was to avoid changing the functionality between this and master. That said I don't have any strong opinions